doc/source/whatsnew/v0.18.2.txt

```diff
@@ -422,3 +422,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+
+- Bug where ``pd.read_gbq()`` could throw ``ImportError: No module named discovery``  (:issue:`13454`)
```

Comment:
The dependency did not change. We don't need to make any changes to install.rst. The google api python client is still installed using `pip install google-api-python-client` . 

In google-api-python-client, a module was renamed from apiclient to googleapiclient to reduce the chance of a naming conflict with another package